geometry_msgs/Point[] points
bool[]                touching
